pyramid_orb/api.py

```diff
@@ -159,7 +159,7 @@ def process(self, request):
             output = {}
             for schema in schemas:
                 dbname = schema['dbname']
-                schema['urlRoot'] = request.host_url + request.path.rstrip('/') + '/' + dbname
+                schema['urlRoot'] = request.path.rstrip('/') + '/' + dbname
```

Comment:
Why are we changing this?  This was specifically to support CORS.  We should come up with a way to support both (look at the requesting path and determine if it needs to be the full url or not)